docs/docs/cli_run.md

```diff
@@ -45,4 +45,29 @@ prancer scenario-arm-pass --db=NONE
 | scenario-arm-pass |  Name of the collection on which you want to run the compliance. |
 | --db=None | It defines that the prancer have to find the collection of snapshot and mastertest files from local filesystem. |
 
-- If you doesn't defines the `--crawler` or `--compliance` then it runs both proceesses. First it crawls the collection and generates the snapshot file.After it runs the compliance on it and generates the output file.
\ No newline at end of file
+- If you doesn't defines the `--crawler` or `--compliance` then it runs both proceesses. First it crawls the collection and generates the snapshot file.After it runs the compliance on it and generates the output file.
+
+## 4) Run Specific compliance on all resources
+
+```
+prancer scenario-arm-pass --db=NONE --mastertestid `mastertestid_of_master_compliance_test` --mastersnapshotid `generated_mastersnapshotid`
```

Comment:
@harshilswan 
This is not correct
this command will run the specific compliance on a specific snapshot id
you should remove `--mastersnapshotid`

Reply:
I have fixed it